controllers/articlesController.js

```diff
@@ -102,15 +91,15 @@ export default class PostController {
             status: CREATED
           });
         }
-        return res.status(GONE).json({
+        return res.status(OK).json({
           message: "Error while bookmarking the article",
           status: INTERNAL_SERVER_ERROR
```

Comment:
@Musigwa Why this response has two different status code? in the `res.body.status` is `INTERNAL_SERVER_ERROR` and the `res.status` is `OK`. I think the status code has to be the same for a single response. In addition, this line is not tested.

Reply:
I think this is an error coming from the past PRs, let me change it and I'll let you know when I'm done.